project-generator/src/main/resources/io/syndesis/project/converter/templates/pom.xml.mustache

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
I've just submitted #414 which properly uses pom properties for other dependencies, but I still think we should directly add the version to the connectors dependencies declarations.